src/main/app/case/to-api-format.ts

```diff
@@ -218,6 +218,13 @@ const fields: ToApiConverters = {
   coApplicant2StatementOfTruth: data => ({
     coApplicant2StatementOfTruth: checkboxConverter(data.coApplicant2StatementOfTruth),
   }),
+  whoIsFinancialOrderFor: data => ({
+    applicant1FinancialOrderFor: data.applyForFinancialOrder !== YesOrNo.NO ? data.whoIsFinancialOrderFor : [],
```

Comment:
imo it's better to use === YesOrNo.Yes since if applyForFinancialOrder is undefined we wouldn't want whoIsFinancialOrderFor to be set anyway

Reply:
it wouldnt get called unless the field whoIsFinancialOrderFor is being passed as a property to send to the backend which is always tagged with applyForFinancialOrder as its on the same page. im happy to change it tho.

Reply:
yeah fair, I guess it doesn't matter either way then. I think === looks nicer, but I don't mind since it's just preference